src/menus/FileMenus.cpp

```diff
@@ -558,7 +558,7 @@ auto FileMenu()
          // pull it out
          // and put it in the Audacity menu for us based on its ID.
          /* i18n-hint: (verb) It's item on a menu. */
-         Command( wxT("Exit"), XXO("E&xit"), OnExit,
+         Command( wxT("Exit"), XXO("Quit Audacity"), OnExit,
```

Comment:
Please keep the accelerator key (letter after &). The way you test them (on Windows and Linux) is by pressing alt, at which point the accelerators get underlined:

![image](https://github.com/user-attachments/assets/1cb42568-e875-43ec-88cf-cb2da986c155)

You can make any character the accelerator as long as it doesn't overlap with any of the other ones shown in the same menu.

Reply:
Should the accelerator key in this case be Q?

Reply:
that looks about right.